bin/sandbox

```diff
@@ -93,12 +93,12 @@ fi
 
 unbundled bin/rails db:drop db:create
 
-echo "~~~> Running the solidus:install generator"
-unbundled bin/rails generate solidus:install \
-  --auto-accept \
-  $@
+# echo "~~~> Running the solidus:install generator"
+# unbundled bin/rails generate solidus:install \
+#   --auto-accept \
+#   $@
 
-echo "
-🚀 This app is intended for test purposes. If you're interested in running
-🚀 Solidus in production, visit:
-🚀 https://guides.solidus.io/developers/getting-started/first-time-installation"
+# echo "
+# 🚀 This app is intended for test purposes. If you're interested in running
+# 🚀 Solidus in production, visit:
+# 🚀 https://guides.solidus.io/developers/getting-started/first-time-installation"
```

Comment:
I think that changes are not intentional, right? :thinking:

Reply:
Definitely not intentional, thanks for catching it!

It was needed in order to test the generator, so some food for thought, might make sense to back port the application generation cache we set up in solidus_stripe.